cpa/tester/policies/helpers/utils/utils.rego

```diff
@@ -15,3 +15,6 @@ test_to_set_set = utils.to_set({1,2,3}) == {1,2,3}
 test_get_element_name_string = utils.get_element_name("hello") == "hello"
 test_get_element_name_object = utils.get_element_name({"world": {}}) == "world"
 test_get_element_name_invalid_object = true if { not utils.get_element_name({"k1": "v1","k2":"v2"}) }
+
+test_is_parameterized_expression_true = utils.is_parameterized_expression("test<< parameters.test >>") == true
```

Comment:
test also with the `pipeline` variant. Its not so much that I am not confident in the regexp, but that we should capture its expected behavior in case it gets modified in the future.